book/routing.rst

```diff
@@ -223,7 +146,7 @@ A basic route consists of just two parts: the ``path`` to match and a
         class MainController extends Controller
         {
             /**
-             * @Route("/")
+             * @Route("/", name="_welcome")
```

Comment:
Also, I think we should choose a different URL here - like `/it-works` - using `/` is not a *great* routing example.